pkg/api/admin/openshiftcluster.go

```diff
@@ -112,15 +112,22 @@ const (
 	VMSizeStandardD16sV3 VMSize = "Standard_D16s_v3"
 	VMSizeStandardD32sV3 VMSize = "Standard_D32s_v3"
 
-	VMSizeStandardE4sV3  VMSize = "Standard_E4s_v3"
-	VMSizeStandardE8sV3  VMSize = "Standard_E8s_v3"
-	VMSizeStandardE16sV3 VMSize = "Standard_E16s_v3"
-	VMSizeStandardE32sV3 VMSize = "Standard_E32s_v3"
+	VMSizeStandardE4sV3   VMSize = "Standard_E4s_v3"
+	VMSizeStandardE8sV3   VMSize = "Standard_E8s_v3"
+	VMSizeStandardE16sV3  VMSize = "Standard_E16s_v3"
+	VMSizeStandardE32sV3  VMSize = "Standard_E32s_v3"
+	VMSizeStandardE64isV3 VMSize = "Standard_E64is_v3"
+	VMSizeStandardE64iV3  VMSize = "Standard_E64i_v3"
```

Comment:
VMTypes added to preview api as well.  Necessary to perform actions on vm types.